skimage/measure/_marching_cubes_lewiner.py

```diff
@@ -119,11 +126,11 @@ def marching_cubes(volume, level=None, *, spacing=(1., 1., 1.),
     if method == 'lewiner':
         return _marching_cubes_lewiner(volume, level, spacing,
                                        gradient_direction, step_size,
-                                       allow_degenerate, use_classic=False)
+                                       allow_degenerate, use_classic=False, mask=mask)
     elif method == 'lorensen':
         return _marching_cubes_lewiner(volume, level, spacing,
                                        gradient_direction, step_size,
-                                       allow_degenerate, use_classic=True)
+                                       allow_degenerate, use_classic=True, mask=mask)
     elif method == '_lorensen':
```

Comment:
When mask is not None, may be add a warning in this case stipulating that mask is not supported with the '_lorensen' method.

Reply:
That is a good idea